### DISCUSSION TITLE
Add Exists to Validation block

### DISCUSSION THREAD

**Discussion by jchamb** [postmanlabs/postman-flows#160] [Ideas]
Add ability to validate a property exists in the validation block. I know there are some work arounds for this, but for simplicity within the interface would be nice to explicitly define that that the validation is testing for existence.

**Comment by samay-v** [comment]
That makes a lot of sense, thanks @jchamb 
Just curious though, what are the work-arounds you use most often for this?

**Reply by jchamb** [reply]
not the greatest or most complete solution, but basically when it does exist and we know the type validate that quickly with the regex option to match type or a enum options etc. Definitely has its limits, but does work for giving your the true false out.
